app/scripts/migrations/070.test.js

```diff
@@ -0,0 +1,84 @@
+import migration70 from './070';
+
+describe('migration #70', () => {
+  it('should update the version metadata', async () => {
+    const oldStorage = {
+      meta: {
+        version: 69,
+      },
+      data: {},
+    };
+
+    const newStorage = await migration70.migrate(oldStorage);
+    expect(newStorage.meta).toStrictEqual({
+      version: 70,
+    });
+  });
+
+  it('should rename NotificationController', async () => {
```

Comment:
Should add a test for when there is no `NotificationController` to rename as well. For users that are starting up MetaMask instances that haven't run/been updated since before ti was added.

Reply:
Done 👍